docs/pages/access-controls/access-monitoring.mdx

```diff
@@ -0,0 +1,208 @@
+---
+title: Getting Started with Access Monitoring
+description: Learn how to use Access Monitoring.
+---
+
+
+
+Teleport Access Monitoring allows to query and analyze the audit events generated by Teleport.
+It provides a SQL interface to query the audit logs and generate reports.
+Access Monitoring is a powerful tool to understand and analyze the access patterns in your Teleport cluster.
+This guide will help you understand how to use Access Monitoring to monitor access to resources in your Teleport cluster.
+
+
+## Prerequisites
+- Teleport Cloud cluster v14.2.2 or later.
+
+## Access Monitoring RBAC Permissions
+
+To access the Access Monitoring interface, a user must have a role that allows
+`list`, `read` and `use` verbs on the `audit_query` and `audit_query` resources.
+By default the built-in `auditor` role has these permissions or you can create a custom role with these permissions e.g:
+```yaml
+kind: role
```

Comment:
Missing `version`?

Reply:
The version seems to be not necessary. Teleport Will set the default  current version during creation.